include/revng/Support/FunctionTags.h

```diff
@@ -121,6 +121,30 @@ class Tag : public DynamicHierarchy<Tag> {
     auto Filter = [this](GlobalVariable &G) { return isExactTagOf(&G); };
     return make_filter_range(M->globals(), Filter);
   }
+
+  auto functions(const llvm::Module *M) const {
+    using namespace llvm;
+    auto Filter = [this](const Function &F) { return isTagOf(&F); };
+    return make_filter_range(M->functions(), Filter);
+  }
+
+  auto exactFunctions(const llvm::Module *M) const {
+    using namespace llvm;
+    auto Filter = [this](const Function &F) { return isExactTagOf(&F); };
+    return make_filter_range(M->functions(), Filter);
+  }
+
+  auto globals(const llvm::Module *M) const {
+    using namespace llvm;
+    auto Filter = [this](const GlobalVariable &G) { return isTagOf(&G); };
+    return make_filter_range(M->globals(), Filter);
+  }
+
+  auto exactGlobals(const llvm::Module *M) const {
+    using namespace llvm;
+    auto Filter = [this](const GlobalVariable &G) { return isExactTagOf(&G); };
+    return make_filter_range(M->globals(), Filter);
+  }
```

Comment:
Can you please interleave these with the non-`const` counterparts? I feel like it reduces the probability of them going out of sync.